app/src/main/java/org/oppia/app/home/HomeActivityPresenter.kt

```diff
@@ -21,26 +21,26 @@ class HomeActivityPresenter @Inject constructor(private val activity: AppCompatA
     setUpNavigationDrawer()
     if (getHomeFragment() == null) {
       activity.supportFragmentManager.beginTransaction().add(
-        R.id.home_fragment_placeholder,
+        R.id.home_activity_home_fragment_placeholder,
         HomeFragment(),
         TAG_HOME_FRAGMENT
       ).commitNow()
     }
   }
 
   private fun setUpNavigationDrawer() {
-    val toolbar = activity.findViewById<View>(R.id.home_activity_toolbar) as Toolbar
+    val toolbar = activity.findViewById<View>(R.id.home_activity_home_toolbar_toolbar) as Toolbar
```

Comment:
This is not making much sense. I want to defer this to @rt4914. It can just be renamed as `home_activity_toolbar`. @rt4914 please let me know.

Reply:
<parent_file_name>_<view_name>_<widget_type>